tests/data/res_tests/resolved/resolved_complex.json

```diff
@@ -0,0 +1,79 @@
+{
+    "apply2": {
+        "process_id": "apply",
+        "arguments": {
+            "data": {
+                "from_node": "valid1_load7"
+            },
+            "process": {
+                "process_graph": {
+                    "nested1_w2_add1": {
+                        "process_id": "add",
+                        "arguments": {
+                            "x": {
+                                "from_parameter": "x"
+                            },
+                            "y": {
+                                "from_parameter": "context"
+                            }
+                        }
+                    },
+                    "nested1_w3_add1": {
+                        "process_id": "add",
+                        "arguments": {
+                            "x": {
+                                "from_parameter": "context"
+                            },
+                            "y": {
+                                "from_parameter": "context"
+                            }
+                        }
+                    },
+                    "nested1_add4": {
+                        "process_id": "add",
+                        "arguments": {
+                            "x": {
+                                "from_node": "nested1_w2_add1"
+                            },
+                            "y": {
+                                "from_node": "nested1_w3_add1"
+                            }
+                        },
+                        "result": true
+                    }
+                }
+            },
+            "context": 0
```

Comment:
I don't think that passing parameters in like this is currently supported by https://github.com/Open-EO/openeo-processes-dask/blob/main/openeo_processes_dask/process_implementations/core.py#L16 - would be great if you could double-check this!

Reply:
This isn't an artifact of resolving. This is just how explicit parameter values are passed by processes with sub-graphs (like apply)

```
{
  "id": "valid_load_small",
  "parameters": [],
  "process_graph": {
    "load7": {
      "process_id": "load_collection",
      "arguments": {
        "id": "boa_sentinel_2",
        "spatial_extent": {
          "west": 16.200432580649483,
          "east": 16.58041390646924,
          "south": 48.04835633006704,
          "north": 48.330435665810796
        },
        "temporal_extent": [
          "2020-07-01T00:00:00Z",
          "2020-07-03T00:00:00Z"
        ],
        "bands": [
          "B02"
        ],
        "properties": {}
      }
    },
    "apply1": {
      "process_id": "apply",
      "arguments": {
        "data": {
          "from_node": "load7"
        },
        "process": {
          "process_graph": {
            "add1": {
              "process_id": "add",
              "arguments": {
                "x": {
                  "from_parameter": "x"
                },
                "y": {
                  "from_parameter": "context"
                }
              },
              "result": true
            }
          }
        },
        "context": 1
      }
    },
    "save2": {
      "process_id": "save_result",
      "arguments": {
        "data": {
          "from_node": "apply1"
        },
        "format": "GTIFF"
      },
      "result": true
    }
  }
}
```
This is a process graph that I threw together quickly in the web-editor (without any UDPs) and the context is in there as well.

I also think it must be supported as I have run the graph (resolved_complex.json) successfully before and if it somehow isn't we should work on supporting it.

Reply:
Right, not related to UDPs, gotcha!
Hmm. I'm not sure tbh, would have to try it out. I'll create an issue to look into it when we have time!